### ISSUE TITLE
Update PNTS Sharing to Organization

### ISSUE THREAD

**Issue by andrewshensky** [cityofaustin/atd-data-tech#7805]
Users from PIO and PWD were unable to access new PNTS web app. I think best solution would be to make web app available at the organization level to eliminate any accessibility issues.

**Comment by jaime-mckeown** [comment]
I found this in new without an assignee.

**Comment by andrewshensky** [comment]
@jaime-mckeown Thanks for finding this one.
